index.markdown

```diff
@@ -13,52 +16,20 @@ layout: default
-
-## Is this a road specific run group? 
-We plan on scheduling road, track, and trail group runs. And there may be some members training for triathlons!
-
-## How do I find out about run meet ups?
-Join us on our Slack space!
-
-## Do I have to pay to join WRC? 
-Joining WRC is free.  We plan to offer some optional gear, such as uniforms, at cost to members. Ultimately, we aim to be transparent about where money we may collect for specific events or reasons is going.
-
-## How can I help? 
-Inviting a friend to WRC, organizing ad-hoc runs in #run-meets in Slack while we get off the ground, and starting conversations in other WRC Slack channels is a great start. Beyond that, areas that will likely need help:
-- Creating a social media presence or maybe a website or listserv
-- Onboarding new members (e.g. adding to Slack and making sure new folks have friendly faces to welcome them at their first run)
-- Planning monthly programming in 2023
-- Organizing Slack channels, adding more fun emojis, starting conversation, and norming
-- Outreach
-- Writing community guidelines / code of conduct for slack
-- Leadership / Strategy 
-If you are interested in any of these please let us know on Slack or email. 
-
-## I have feedback and questions. Who do I ask? 
-Please share feedback, ideas, and questions in whatever medium is most comfortable to you. As a starting point, feel free to post in the general channel on WRC Slack or via Slack DM if you prefer a more private conversation.
+# Logistics
+Our club is what we build together. Since we are new, we don't have 100% of things planned perfectly and are trying things to see what works (let's learn, shift, and grow together) <3 This is what we have planned in 2023:
+- Our club is free to join (we don’t have any costs yet). After you sign up we'll add you to our Slack workspace where we coordinate and communicate, and to our mailing list where we send one email monthly.
+- We host weekday runs in Oakland and weekend runs at rotating locations around the East Bay. Once you join please do make an effort to come on out and join us in person! 
+- Our Slack forum is active with planning ad hoc meet-ups, discussing training, postpartum/injury recovery journeys, volunteering/community engagement opportunities, gear swap & more!
 
+In the future we could incorporate as a non-profit if we think the structure will help us or charge a small membership fee if we have costs.
```

Comment:
I don't want to have a membership fee, but I'm fine charging for specific events or causes.